code/__HELPERS/time.dm

```diff
@@ -9,7 +9,7 @@
 
 ///returns the current IC station time in a world.time format
 /proc/station_time(display_only = FALSE, wtime=world.time)
```

Comment:
```suggestion
/proc/station_time(wtime = world.time)
```
We don't use display_only anywhere, and it's no longer using the timezoneoffset so not required. station_time should return a consistent time no matter what anyways, and now it does.